docs/images/siddhi-logo-w.svg

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year

Reply:
Fixed with commit: `Fix some suggested issues in PR#1.` 8a89f22